amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseManagementFactory.java

```diff
@@ -549,7 +554,8 @@ public DynamoDBLeaseManagementFactory(final KinesisAsyncClient kinesisClient,
             final long listShardsCacheAllowedAgeInSeconds, final int cacheMissWarningModulus,
             final long initialLeaseTableReadCapacity, final long initialLeaseTableWriteCapacity,
             final HierarchicalShardSyncer deprecatedHierarchicalShardSyncer, final TableCreatorCallback tableCreatorCallback,
-            Duration dynamoDbRequestTimeout, BillingMode billingMode, Collection<Tag> tags, LeaseSerializer leaseSerializer,
+            Duration dynamoDbRequestTimeout, BillingMode billingMode, final boolean deletionProtectionEnabled,
```

Comment:
Is there any chance that someone could be using this to crate directly? Would this be code breaking for us to get rid of the builder with just the current ones, and if so I would recommend having another method that includes deletionProtectionEnabled, and then have this one call that with it being false.

Reply:
This class is annotated with `@KinesisClientInternalApi` which means `Any class/method/variable marked with this annotation is subject to breaking changes between minor releases.`